Find a functionally equivalent code:android.graphics.Paint.setPathEffect:COMMENT
Method Modifier: public      
Comment:/**
 * Set or clear the patheffect object.
 * <p />
 * Pass null to clear any previous patheffect.
 * As a convenience, the parameter passed is also returned.
 *
 * @param effect May be null. The patheffect to be installed in the paint
 * @return       effect
 */

@@ -1,5 +1,5 @@
 {
-    int effectNative = 0;
+    long effectNative = 0;
     if (effect != null) {
         effectNative = effect.native_instance;
     }

